### DISCUSSION TITLE
Chat Commands.

### DISCUSSION THREAD

**Discussion by RanarTheWrathful** [Taureon/aps-plus-plus#217] [Suggestions]
For either settings or for those with the token.
I can give some examples later on if this is of interest.

**Comment by Trioplane** [comment]
You can find a chat command addon at [#community-addons > Chat Commands](https://discord.com/channels/1004907608018264094/1148897802877927425/threads/1150320798465282110/1150320798465282110). We probably don't need to add more.

**Comment by Taureon** [comment]
chat commands need to be intengrated better into aps++
it would make it easier to add a bunch of commands essentials

**Comment by archbtww** [comment]
i agree

**Comment by Trioplane** [comment]
So like `StatusEffect` but for chat commands?

**Comment by Taureon** [comment]
yea
